File: Aula25_10_12_2020/Exercícios/Cs/LetraB.cs
using System;

namespace Cs
{
    class letraB
    {
        static void Main(string[] args)
        {
            int[] matrizA = new int[8], matrizB = new int[8];

            for (int i = 0; i < matrizA.Length; i++)
            {
                Console.Write($"Informe o elemento {i}: ");
                matrizA[i] = Convert.ToInt32(Console.ReadLine());
                
                matrizB[i] = 3 * matrizA[i];
            }
            Console.WriteLine("----------------------------------------------");
            for (int i = 0; i < matrizA.Length; i++)
            {                                                           //\t é como se fosse um tab para exibir
                Console.WriteLine($"Elemento {i} = Vetor A: {matrizA[i]} \t Vetor B: {matrizB[i]}");
            }
            
        }
    }
}

File: Aula25_10_12_2020/Exercícios/Py/letraB.py
matrizA = [1, 1, 1, 1, 1, 1, 1, 1]
matrizB = [1, 1, 1, 1, 1, 1, 1, 1]

for i in range(0, 8, 1):
    matrizA[i] = int(input(f"Informe o elemento {i}: "))
    matrizB[i] = matrizA[i] * 3
    
for x in range(0, 8, 1):
    print(f"Índice {x} = Vetor A: {matrizA[x]} e Vetor B: {matrizB[x]}")